lib/plenario_web/controllers/web/page_controller.ex

```diff
@@ -8,18 +8,7 @@ defmodule PlenarioWeb.Web.PageController do
   alias PlenarioAot.{AotData, AotMeta}
 
   def assign_s3_path(conn, _opts) do
-    assign(
-      conn,
-      :s3_asset_path,
-      Map.get(
-        %{
-          "dev.plenar.io": "https://s3.us-east-2.amazonaws.com/plenario2-assets-staging",
-          "plenar.io": "https://s3.us-east-2.amazonaws.com/plenario2-assets"
-        },
-        conn.host,
-        "https://s3.us-east-2.amazonaws.com/plenario2-assets-dev"
-      )
-    )
+    assign(conn, :s3_asset_path, "https://s3.amazonaws.com/plenario2-assets")
```

Comment:
The images couldn't be found because the `plenario2-assets` bucket didn't exist. When I created the bucket it had a new naming scheme that omitted the `us-east-2`. Instead of trying to convenience the developer by pointing to different buckets dynamically, I'd rather just overwrite the string manually and manage a single bucket.